airbyte-integrations/connectors/source-shopify/unit_tests/integration/api/authentication.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+import json
+
+from airbyte_cdk.test.mock_http import HttpMocker, HttpRequest, HttpResponse
+from airbyte_cdk.test.mock_http.request import ANY_QUERY_PARAMS
+from airbyte_cdk.test.mock_http.response_builder import find_template
+
+_ALL_SCOPES = [
```

Comment:
This is hard, to maintain in the future, because we already have the list of scopes, that could be leveraged from the actual place they maintain (single place) https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-shopify/source_shopify/scopes.py#L14, let's have the method for this tests that parses this object and returned the `list of strings`.  

```python
get_list_of_scopes_from_scopes_mapping(SCOPES_MAPPING) -> List[str]:
   # parse the SCOPES_MAPPING object
   parsed_mapping_as_list_of_strings = ....
   return List[parsed_mapping_as_list_of_strings]
```